test/pobox_heir_SUITE.erl

```diff
@@ -0,0 +1,111 @@
+-module(pobox_heir_SUITE).
+-include_lib("common_test/include/ct.hrl").
+-compile(export_all).
+
+all() -> [
+    with_local_registered_name_owner,
+    with_global_registered_name_owner,
+    with_via_registered_name_owner,
+    owner_crashes_heir_takes_over,
+    heir_dies_then_owner_dies
+].
+
+
+with_local_registered_name_owner(_Config) ->
+    Self = self(),
+    Ref = make_ref(),
+    PreviousOwnerPid = erlang:spawn(fun() ->
+        {ok, Box} = pobox:start_link({local, ?MODULE}, [{heir, Self}, {heir_data, Ref}]),
+        Self ! Box,
+        throw("crash")
+                                    end),
```

Comment:
that little `end` is really hanging away far right :P

Reply:
Sorry that's my IDE being stupid. I will fix it in the final PR.